diagnostic.md

```diff
@@ -7,36 +7,36 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git@github.com:beckyd4234/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
-
-<!-- Remove this comment and place your answer here. -->
+git status
```

Comment:
Inorrect. Using git status frequently is a good habit in general but the answer is to make a new branch.

You never want to commit to master, so the first step after cloning is always to make a new branch to work in.